src/bin/frontendlauncher

```diff
@@ -110,6 +114,20 @@ do
         have_version=yes
     fi
 done
+
+# If we have a two digit version number, find the latest patch associated
+# with that two digit version. 
+if [[ $version =~ ^[0-9]+.[0-9]+$ ]]; then
+    for patch in 6 5 4 3 2 1 0; do
```

Comment:
Have we never gone beyond '6' as a patch number? Or are you anticipating in the future we won't ever go beyond that?

Reply:
We've never gone beyond 4. There is logic earlier in the script that assumes we won't go beyond 4, so decided to go to 6 to give us some breathing room. I should have updated that logic as well. I've just increased the valid patch numbers in the entire script to 9, so we should have a large cushion.